src/main/java/org/jabref/preferences/PreferencesService.java

```diff
@@ -11,4 +12,10 @@
     KeyBindingRepository getKeyBindingRepository();
 
     void storeJournalAbbreviationPreferences(JournalAbbreviationPreferences abbreviationsPreferences);
+
+    FileDirectoryPreferences getFileDirectoryPreferences();
+
+    String get(String key);
```

Comment:
Please don't add generic methods like this! The idea was that the `PreferencesService` provides a typed interface for the preferences. Thus here it would be `set/getWorkingDirectory` with `nio.path` as argument/return value.